stix_shifter_modules/alertflex/configuration/alertflex_config.json

```diff
@@ -0,0 +1,55 @@
+{
+    "connection": {
+        "type": {
+            "default": "alertflex"
```

Comment:
"type": {
            "default": "alertflex",
            "displayName": "Alertflex(OR whatever you want to choose as display name in the UI)",
            "type": "connectorType"
        }